Gordon360/Services/AccountService.cs

```diff
@@ -170,6 +172,7 @@ public AccountViewModel GetAccountByUsername(string username)
 
             if (hall is not null) accounts = accounts.Where(a => a.Hall.StartsWithIgnoreCase(hall));
             if (classType is not null) accounts = accounts.Where(a => a.Class.StartsWithIgnoreCase(classType));
+            if (preferredClassYear is not null) accounts = accounts.Where(a => a.PreferredClassYear.StartsWithIgnoreCase(preferredClassYear));
```

Comment:
Class year is not a fuzzy search. Please use `==` instead of `.StartsWithIgnoreCase(`